src/pkg/utils/exec/exec.go

```diff
@@ -207,3 +209,17 @@ func GetOSShell(shellPref types.ZarfComponentActionShell) (string, string) {
 func IsPowershell(shellName string) bool {
 	return shellName == "powershell" || shellName == "pwsh"
 }
+
+// ExitOnInterrupt catches an interrupt and exits with ErrorCode
+func ExitOnInterrupt(ErrorCode int, ErrorMessage string) chan os.Signal {
```

Comment:
```suggestion
func ExitOnInterrupt() chan os.Signal {
```
Per the below this likely can be simplified to just use a reusable error